docs/2.1. APIs - Client Side Implementation.md

```diff
@@ -4,7 +4,7 @@ _(c) AMWA 2017, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
 
 ## Conforming to Schemas
 
-The Connection Management API schemas define sets of parameters for each supported transport type; these are also described in pages in the [Behaviour](4.0.%20Behaviour.md) section of these documents. The schemas define which parameters are always present (core), and which are feature-dependent; a client can determine what a Device supports by getting its `/constraints` resource, and SHOULD parse this resource in all cases as a Device might have a constrained range or enumeration of supported values, even for core paramters.
+The Connection API schemas define sets of parameters for each supported transport type; these are also described in pages in the [Behaviour](4.0.%20Behaviour.md) section of these documents. The schemas define which parameters are always present (core), and which are feature-dependent; a client can determine what a Device supports by getting its `/constraints` resource, and SHOULD parse this resource in all cases as a Device might have a constrained range or enumeration of supported values, even for core paramters.
```

Comment:
There's a typo in the last word of the paragraph.
But more importantly, I'm not sure this is true as now written. The schemas _do_ define the parameters for each supported transport type. But they _don't_ define the _sets of parameters_ nor which ones are core. That, unfortunately, is _only_ done by the documentation in the relevant subsections.